html/status.html

```diff
@@ -606,6 +659,9 @@
 			getData();
 			reloadPage();
 		});
+
```

Comment:
LGTM. Can you remove those lines please ?

Reply:
@catalinii done :)